CSS/style.css

```diff
@@ -7,15 +7,19 @@
 }
 
 .app {
-    
     height: 100vw;
```

Comment:
It's not a common thing to use `height` property with `vw` (viewport width), It is used for width. What you can do is remove this line. Here and I'll show you common approach to use `height` below.